OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -75,6 +75,8 @@ public bool InterpretCommand(S server, Connection conn, Session.Client client, s
 			if (!ValidateCommand(server, conn, client, cmd))
 				return false;
 
+			var players = new MapPlayers(server.Map.PlayerDefinitions).Players;
```

Comment:
You should cache the MapPlayers object on Server (and update it whenever there is a change map order) instead of reparsing it in all of the server traits.